### DISCUSSION TITLE
Congrats.

### DISCUSSION THREAD

**Discussion by st0le** [pomdtr/sunbeam#45] [General]
I just want to thank you for the work. This is really great tool and one of the only "quick hack" tools that works well on Windows. I'm hoping there will be a frontend (GUI) someday.

Replacing my Slapdash (which seems abandoned) with sunbeam.

**Comment by pomdtr** [comment]
Thanks for the kind words! 

I'm still experimenting with guis. I have one on my mac (https://github.com/pomdtr/sunbeam-raycast), and I think the next one will probably a pwa to be able to run sunbeam from my phone.

I'm currently focusing on sunbeam scripts running on serverless platforms (ex: `sunbeam fetch https://deno-oak-pomdtr.deno.dev`), as I feel these kind of scripts will be necessary for cross platform guis (ex: https://github.com/pomdtr/deno-oak-sunbeam).

Another ideas is to distribute sunbeam as a universal launcher for the webs.

As you can see I'm a bit overwhelmed by the amount of thing I want to build around this project 😅.
